sass/mixins/_mixins.labels.scss

```diff
@@ -14,3 +14,14 @@
     }
   }
 }
+
+@mixin label--affiliate() {
+  @include text-helper($label--affiliate-font-id, $label--affiliate-size--desktop);
```

Comment:
remove `,$label--affiliate-size--desktop` or use 

`@include text-helper($label--affiliate-font-id, $label--affiliate-size--mobile);` and remove next line